### ISSUE TITLE
[ally.fix.pointerFocusParent] keyboard shown on every tap in iOS10

### ISSUE THREAD

**Issue by rodneyrehm** [medialize/ally.js#153]
*Concerning*: [ally.fix.pointerFocusParent](https://github.com/medialize/ally.js/blob/master/docs/api/fix/pointer-focus-parent.md)
*User Agent:* Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Version/10.0 Mobile/14B100 Safari/602.1
*ally.js version*: 1.3.2
---

## Steps to reproduce the problem

1. open the [demo page](https://allyjs.io/api/fix/pointer-focus-parent.example.html) in iOS10 Safari
2. tap on the fixed link
3. notice the OSK (OnScreenKeyboard) showing up

## Comments

The fix is still necessary for desktop WebKit

**Comment by rodneyrehm** [comment]
The problem seems to be running `ally/supports/supports` during the handling of `touchstart`. For some reason iOS10 gets stuck on triggering the OSK on every `touchstart`. The following will show the OSK on every tap, even though the handler is removed after the first execution:

```js
var supports = require('ally.js/supports/supports');
function run() {
  console.log("running supports tests");
  supports();
  document.removeEventListener('touchstart', run, true);
}

document.addEventListener('touchstart', run, true);
```

**Comment by rodneyrehm** [comment]
The problem is caused by a combination of tests involving an iframe: https://jsbin.com/taviso/1/edit?js,output - it occurs on iOS10.1.1, but seems to be fixed in iOS10.2.0

**Comment by rodneyrehm** [comment]
We now have multiple supports tests involving iframes, so reordering the tests doesn't works. This has been fixed in iOS 10.2.0. The problem can be worked around by running `ally/supports/supports` *before* the first user interaction. Therefore closing this as wontfix.
